apps/fishing-map-e2e/src/e2e/sidebar/save-workspace.cy.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  disablePopups,
+  getWorkspace,
+  waitForMapLoadTiles,
+  waitForSidebarLoaded,
+} from '../../support/app.po'
+import { WORKSPACE_NAME } from '../../constants/workspace'
+
+describe('Save a workspace', () => {
+  before(() => {
+    cy.login(Cypress.env('apiAuthUser'), Cypress.env('apiAuthPass'))
+  })
+  beforeEach(() => {
+    disablePopups()
+    cy.visit('/')
+    waitForSidebarLoaded()
+    waitForMapLoadTiles()
+  })
+
+  it('Should create a workspace', () => {
+    cy.getBySel('save-workspace-button').click()
+    cy.getBySel('create-workspace-inpunt').clear()
+    cy.getBySel('create-workspace-inpunt').type(WORKSPACE_NAME)
+    cy.getBySel('create-workspace-button').click()
+    getWorkspace().should('contain', WORKSPACE_NAME)
+  })
+
+  it('Should delete a workspace', () => {
```

Comment:
Tests should be independent and not depend from others. This one is depending on the preious test that creates the workspace. I'd merge both tests into one

Reply:
and as we talked in the meeting the workspaces should be deleted before the test